vimeo-networking/src/main/java/com/vimeo/networking/Configuration.java

```diff
@@ -332,24 +309,16 @@ public Builder(String clientId, String clientSecret, String scope,
          * @param accountStore  (Optional, Recommended) An implementation that can be used to interface with Android's
          *                      <a href="https://goo.gl/QZ7rm">Account Manager</a>
          */
-        @Deprecated
-        public Builder(@Nullable String baseURLString,
-                       String clientId,
-                       String clientSecret,
-                       String scope,
+        public Builder(@NotNull String clientId,
```

Comment:
I agree with all of those points. And it also looks like there is definitely some scope creep (which Kevin says below) if we were to get to all of them.

I would suggest perhaps just doing option (2) for now (since that directly effects a change made in this PR), and then adding the remainder of Howard's comments to the ticket regarding conf builder cleanup.